### PR TITLE
CVSL-1629-Initial appointment When page Changing the order of the rad…

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**
…io buttons

### CODE REVIEW

File: server/enumeration/appointmentTimeType.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 enum AppointmentTimeType {
+  SPECIFIC_DATE_TIME = 'At a specific date and time',
```
Comment (nishanthmahasamudram) [inline server/enumeration/appointmentTimeType.ts:2]: To bring date time option to top.

File: server/views/pages/create/initialMeetingTime.njk [hunk 2]
```diff
@@ -98,7 +98,7 @@
                     {{ govukRadios({
                     id: "appointment-time-type",
                     name: "appointmentTimeType",
-                    value: formResponses.appointmentTimeType or licence.appointmentTimeType,
+                    value: (formResponses.appointmentTimeType or licence.appointmentTimeType) or SPECIFIC_DATE_TIME,
```
Comment (nishanthmahasamudram) [inline server/views/pages/create/initialMeetingTime.njk:101]: To select Date time option default when the user first comes to this screen.

Comment (andrewrlee) [reply server/views/pages/create/initialMeetingTime.njk:101]: is this needed to expand the box?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/pages/create/initialMeetingTime.njk (hunk 2 above)
Comment (nishanthmahasamudram) [reply server/views/pages/create/initialMeetingTime.njk:101]: Yes it is needed.

**Pull request merged**